### DISCUSSION TITLE
Cypress.Cookies.defaults() will be removed. What to do?

### DISCUSSION THREAD

**Discussion by jitenev** [cypress-io/cypress#21990] [Questions and Help]
Tests need to learn session cookies, both between tests within the same spec and between different specs. 
Cypress.Cookies.defaults() -> solved my problem perfectly.
cy.session() -> does not solve this problem.
Does anyone have any ideas on this?

**Comment by jozso39** [comment]
I recommend turning off `testIsolation` to tests, you have to specify it on the top describe of the test:
```
describe(`your test`, { testIsolation: false }, () => {
		it(`this test or step`, () => {
			// test code
		});
	});
	
```

This way, the session will not be cleared between it-s.
There is another problem: the sessions will carry over to another tests. You just have to clear the session in the before hook.
